app/src/main/java/se/nullable/flickboard/ui/Settings.kt

```diff
@@ -674,6 +675,7 @@ enum class Handedness(override val label: String) : Labeled {
 
 enum class LetterLayerOption(override val label: String, val layout: Layout) : Labeled {
     English("English (MessagEase)", EN_MESSAGEASE),
+    Spanish("Spanish (MessagEase)", ES_MESSAGEASE),
```

Comment:
Nit: Please keep the list alphabetic by display name, not language code (DE is also below EN for this reason).

Reply:
fixed in [12cd91f](https://github.com/nightkr/flickboard/pull/57/commits/12cd91fa604c5293965f9f26fea44717858b452a)